src/pages/AppFrame.css

```
@@ -62,23 +62,41 @@ footer {
   padding-top: 60px;
 }
 
+.searchDiv {
+  color: #9fa6b2;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+  letter-spacing: -0.2px;
+}
+
+.searchVal {
+  color: #373740;
+  font-size: 32px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: normal;
+  letter-spacing: -0.2px;
+}
```

겹치는 스타일을 중복 부여할 필요 없이, 해당 요소에 다르게 적용되는 스타일만 부여하시면 됩니다.
```suggestion
.searchDiv {
  color: #9fa6b2;
  font-size: 32px;
  font-style: normal;
  font-weight: 600;
  line-height: normal;
  letter-spacing: -0.2px;
}

.searchVal {
  color: #373740;
}
```